libs/judicial-system/types/src/lib/dates.ts

```diff
@@ -0,0 +1,27 @@
+import addDays from 'date-fns/addDays'
+
+const VERDICT_APPEAL_WINDOW_DAYS = 28
+const FINE_APPEAL_WINDOW_DAYS = 3
+
+const APPEAL_WINDOW_DAYS = 3
+
+const setEndOfDay = (date: Date) => new Date(date.setHours(23, 59, 59, 999))
+
+export const hasDatePassed = (deadline: Date) => Date.now() > deadline.getTime()
+
+export const getIndictmentAppealDeadlineDate = ({
```

Comment:
We generally don't accept objects as parameters. Please consider rewriting this as 

`export const getIndictmentAppealDeadlineDate = (baseDate: Date, isFine?: boolean) => ...`